Platform/QemuBoardPkg/BoardConfig.py

```diff
@@ -23,16 +23,19 @@ def __init__(self, *args, **kwargs):
 
         super(Board, self).__init__(*args, **kwargs)
 
-        EXECUTE_IN_PLACE          = 0
+        # 0: QEMU  1: Simics support ->QSP
+        self.ENABLE_SIMICS        = 0
+
+        EXECUTE_IN_PLACE          = 0 if self.ENABLE_SIMICS != 1 else 1

```

Comment:
why do we need to do execution in place for QSP ?

Reply:
Done removed. It was use initially to skip the codes in !XIP but that was not the proper way. I added the board check flag instead.